docs/src/dev_guide/inputs.md

```diff
@@ -0,0 +1,337 @@
+# Inputs
+
+Inputs used to run the holistic electricity model: [indices](#indices) and [parameters](#parameters). [Indices](#indices) refer to the dimensions using which the rest of the data is organized, such as the number of generation and storage resources, independent power producers, the balancing areas condsidered, the representative days considered etc.
+
+## Indices
+
+### Common Indices
+
+ - **index_y.csv**: Representative simulation years. It consisits of a single row of year values. The file is used to generate two types of indices, `index_y` which is the years for a simulation window and `index_y_fix` which represents the full simulation horizon (does not change).
+
+ - **index_s.csv**: New resource depreciation years represented as a row vector.
+
+ - **index_d.csv**: Representative day and hour (from ReEDS). Each day is represented as `d[0-9][0-9][0-9]` in a row vector.
+
+ - **index_t.csv**: Representative hour within each representative day (from ReEDS) represented as `h[0-9][0-9][0-9]` in a row vector.
+
+ - **index_h.csv**: Customer groups. Example values are `Residential, Commercial, Industrial`.
+
+ - **index_j.csv**: Green technology types. Example values are `biopower, wind-ons, wind-ofs`.
```

Comment:
```suggestion
 - **index_j.csv**: Green technology types. Example values are `biopower, wind-ons, wind-ofs`. This index is related to the `GreenDeveloper` agent and direct purchases of clean electricity by customers through power purchase agreements.
```

Reply:
Could also put this under a `#### Green Developer` heading. It's really most like `index_k_new` or `index_rps`.